pkg/cmd/generate-bazel-extra/main.go

```diff
@@ -48,15 +49,54 @@ func runBuildozer(args []string) {
 	}
 }
 
-func getTestTargets(testTargetSize string) ([]string, error) {
-	if _, ok := testSizeToDefaultTimeout[testTargetSize]; !ok {
-		return nil, errors.New("testTargetSize should be one of {small,medium,large,enormous}")
+// parseQueryXML is used because Go doesn't support parsing XML "1.1".
+// It returns a map where the key is a test size {small,medium,large,enormous} and
+// the value is a list of test targets having that size.
+func parseQueryXML(data []byte) (map[string][]string, error) {
+	targetNameRegex, err := regexp.Compile(`<rule class="go_test" location=".*" name="(.*)">`)
+	if err != nil {
+		return nil, err
+	}
+
+	targetSizeRegex, err := regexp.Compile(`<string name="size" value="(.*)"/>`)
```

Comment:
Note: normally I would want these regexps in some global (immutable, private) variables so they are only instantiated once per run of the program. However we only call this function once so it's fine. Something to think about for the future.